examples/hello-world/hello-service.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+import asyncio
+from datetime import datetime, timedelta
+
+# from yapapi.log import enable_default_logger, log_summary, log_event_repr  # noqa
+
+from yapapi import Golem
+from yapapi.executor.services import Service
+from yapapi.payload import vm
+
+DATE_OUTPUT_PATH = "/golem/work/date.txt"
+POLLING_INTERVAL_SEC = 5
+
+
+class DateService(Service):
+    @staticmethod
+    async def get_payload():
+        return await vm.repo(
+            image_hash="d646d7b93083d817846c2ae5c62c72ca0507782385a2e29291a3d376",
+        )
+
+    async def start(self):
+        self._ctx.run(
+            "/bin/sh",
+            "-c",
+            f"while true; do date > {DATE_OUTPUT_PATH}; sleep {POLLING_INTERVAL_SEC}; done &",
+        )
+        yield self._ctx.commit()
+
+    async def run(self):
+        while True:
+            await asyncio.sleep(POLLING_INTERVAL_SEC)
+            self._ctx.run(
+                "/bin/sh",
+                "-c",
+                f"cat {DATE_OUTPUT_PATH}",
+            )
+
+            future_results = yield self._ctx.commit()
+            results = await future_results
+            print(results[0].stdout.strip())
+
+
+async def main():
+    async with Golem(
+        budget=1.0,
+        subnet_tag="goth",
```

Comment:
Works for me, but let's not forget to change it to "devnet-beta.2" (or maybe don't specify any subnet?)

Reply:
well, maybe we should use the same subnet as the others are using? ... 

perhaps, `examples.utils` should have a constant DEFAULT_SUBNET that the argparser uses as the default for the `--subnet-tag` argument and which a minimal example like this one here could use directly?